cumulus_library/studies/discovery/manifest.toml

```diff
@@ -0,0 +1,12 @@
+study_prefix = "discovery"
```

Comment:
this name is just ok - soliciting options.

Reply:
For context, will this study also maybe be used in the future for other sort of "is your site's data ready for Cumulus Library or do we need to make tweaks" kind of stuff (i.e. it will be a component of site on-boarding)?

Brainstorming: `onboarding`, `data-quality`, `data-qa`, `data-report`, `fhir-quality`, `library-prep`

I dunno. Discovery works for me, though it is lightly generic. But that might be good, depending on how kitchen sink we want it to be able to get.